ORG_STRUCTURE.md

```diff
@@ -0,0 +1,76 @@
+# OOT Structure
+There are many challenges to distributed teams such as miscommunication and
+productivity. That's at OOT we're trying a decentralized, self-management
+system for our teams and it's called a **holacracy**.
+
+## Welcome to Holacracy! 
+
+A [holacracy](https://www.holacracy.org/) is a decentralized, self-management 
+system for running purpose-driven organizations, with reduced hierarchy. 
+Essentially it means that not all decisions go through one person, but that 
+each member of a circle feels empowered to take control over their task and
+responsibilities within the circles domain. Let me explain...
+
+The essential elements are:
+
+#### Circles
+Circles are like project, and each circle has assigned a clear purpose and 
+accountabilities. Each circle has the authority to self-organize internally to 
+best achieve its goals. Circles are connected by "Rep Links", which are people 
+who attend meetings for both their circle and other circles needed to ensure 
+their own alignment with the broader organization’s mission and strategy.
+
+A circle has four core roles:
+    • Lead link – The lead link contains the circle’s purpose and is responsible for the circle’s successes and failures throughout its existence. 
+    • Secretary –  The secretary schedules meetings and makes note of the meeting’s outlook. Important for inter circle communication
+    • Rep Link – The rep link is an elected role used to represent the interests of the circle in other circles
+
+#### Members
+Members are you folks, the maintainers. The number of maintainers is not divided equally between the circles. One person can sit in multiple circles and one circle can (should) have multiple people
+
+Tasks
+A "task" is an issue on github. An issue (called feature implementation when you’re making one on GH) is an attempt to clearly define the scope of the task, responsibilities and accountabilities of the task.  It is composed of 
+    • An objective and description of the task: 
+    • Skills and tools/dependencies  needed
+    • Time estimate
+    • Suggested steps to complete the task
+Tasks are decided in a circle through its decision processes (that it decides) and should be updated regularly in order to achieve the circle’s purpose within the shortest time frame. These tasks should live in the circles board on github.
+The person that fulfills any given task is empowered to make decisions and execute assignments in the way that they think is best within the circles domain.
+
+## OOT Structure Snapshot
+If you have questions regarding certain topics or areas of OOT, here you can
+find who you should ping.
+
+#### Data Team
+- Andrada Pumnea
+- Matteo Guzzo
+
+#### Engineering & DevOps
+- River Honer
```

Comment:
Eep um I would much prefer a link to my github than just my name. I want to change the name of my github anyway, away from my legal name

Reply:
Also @malteserteresa you mentioned in Slack that you put only the people who had their portraits on the website in the docs but River is not on the website. Should we just do a poll in Slack to ask who wants to be in the docs and if we should use our real names or only our Github names?

Reply:
I have deleted the names in the new version of the file. My idea is that we just explain our structure in this file and then in each repo README, people can add themselves if they want and how they want in a "Repository management" section. See this [branch](https://github.com/opt-out-tools/model-online-misogyny/tree/feature/%2370-improve_docs) as an example.